### ISSUE TITLE
Multi-select options

### ISSUE THREAD

**Issue by krhee1026** [hougaard/Simple-Object-Designer#314]
Is there any way to create an option list where multiple options can be selected via SOD?

**Comment by hougaard** [comment]
Yes, field type = "Option" and then open the Field Settings page to specify a comma-separated list of options...

**Comment by krhee1026** [comment]
Sorry, I should have been clearer.  Is it possible to have a list where I can select more than one option?

**Comment by hougaard** [comment]
Nope, that's not a thing in Business Central, there are no fields that'll hold that. 

The other option is to create a feature, but then it gets complicated.

**Comment by krhee1026** [comment]
Thanks, Eric.  I appreciate the answers!  I do love SOD, by the way.  I am not a BC developer, but am the project manager for our recent implementation. It is a great product!
